app/import.go

```diff
@@ -35,6 +37,9 @@ func (a *App) bulkImportWorker(dryRun bool, wg *sync.WaitGroup, lines <-chan Lin
 
 func (a *App) BulkImport(fileReader io.Reader, dryRun bool, workers int) (*model.AppError, int) {
 	scanner := bufio.NewScanner(fileReader)
+	buf := make([]byte, 0, 64*1024)
```

Comment:
Why this size is not `maxScanTokenSize`?

Reply:
My nterpretation of [docs](https://pkg.go.dev/bufio?tab=doc#Scanner.Buffer) : `buf` is the initial buffer to use when scanning by default, if it overflows, the size of `buf` is increased until it reaches to `maxScanTokenSize`. The idea is that to keep buffer small at the beginning.